docs/userguide/index.md

```diff
@@ -12,9 +12,7 @@ parent = "mn_fun_docker"
 
 In the [Introduction](../misc) you got a taste of what Docker is and how it
 works. In this guide we're going to take you through the fundamentals of
-using Docker and integrating it into your environment.
-
-We’ll teach you how to use Docker to:
+using Docker and integrating it into your environment. We’ll teach you how to use Docker to:
```

Comment:
"We'll teach you" -> "You're going to learn" (or "You'll learn", but that's future...)